buildtest/menu/build.py

```diff
@@ -74,19 +74,57 @@ def discover_configs(config_file):
         logger.error(msg)
         sys.exit(msg)
 
+    # return all configs by resolving path, this gets the real canonical path and address shell expansion and user expansion
+    config_files = [resolve_path(config) for config in config_files]
+
     logger.info(f"Found the following config files: {config_files}")
     return config_files
 
 
+def include_file(file_path, white_list_patterns):
+    """Check if file is included based on OR regular expression. If no white_list_patterns
+       provided method will return ``True``. Otherwise it will return the list of files that don't
+       match the regular expression
+
+       Parameters:
+
+       :param file_path: file path to run regular expression upon
+       :type file_path: str, required
+       :param white_list_patterns: the exclude list provided on command line option
+       :type white_list_patterns: list, required
+       :return: Returns True or a list of files that don't match regular expression
+       :rtype: bool or str
+    """
+
+    logger = logging.getLogger(__name__)
```

Comment:
Is there a reason that the logger is instantiated here, instead of for the module?

Reply:
ahh you mean just instantiate once per file as global variable and not call inside each method. I think that makes sense. If that's what you mean i think it makes sense and i agree with your logic.

Reply:
Exactly - that's why we use __name__ for the logger, it refers to the active module.

Reply:
I fixed this now